<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MathOpePanel</class>
 <widget class="QDialog" name="MathOpePanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>120</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>540</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MathOpePanel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QLabel" name="newPlotNameLabel">
     <property name="text">
      <string>New Plot Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="nameFormFst"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="mathOpeLabel">
     <property name="text">
      <string>Math Function</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="3">
    <widget class="QLineEdit" name="newPlotNameEdit"/>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLineEdit" name="mathOpeEdit"/>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="nameFormSnd"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="nameLabelFst">
     <property name="text">
      <string>Plot Name 1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="nameLabelSnd">
     <property name="text">
      <string>Plot Name 2</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QPushButton" name="cancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="yesBtn">
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
